net/browserless_agent/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@graphai/browserless_agent",
+  "version": "0.0.1",
+  "description": "",
+  "main": "lib/index.js",
+  "files": [
+    "./lib"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "eslint": "eslint src --fix",
+    "format": "prettier --write '{src,tests}/**/*.ts'",
+    "doc": "npx agentdoc",
+    "test_run": "node --test --require ts-node/register ./tests/run_*.ts",
+    "test": "node --test  -r tsconfig-paths/register --require ts-node/register ./tests/test_*.ts",
+    "ci": "yarn run format && yarn run eslint && yarn run test && yarn run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/receptron/graphai-agents.git"
+  },
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/receptron/graphai-agents/issues"
+  },
+  "homepage": "https://github.com/receptron/graphai-agents#readme",
+  "devDependencies": {
+    "undici": "^6.21.1",
+    "@receptron/test_utils": "1.0.1",
+    "@graphai/agent_filters": "0.2.2"
+  },
+  "dependencies": {
+    "graphai": "1.0.0"
```

Comment:
please remove it because graphai also defined in root and  it is always used together except under specific conditions.

Reply:
Thank you for the comments! Since other agents rely on types from an older version of graphai, updating everything to the latest version causes type errors, and the tests fail. I've included it here because the scope of the fix is broad.